src/test/unit/goCenterClientTest.ts

```diff
@@ -0,0 +1,65 @@
+import { GoCenterClient } from '../../main/goCenterClient/GoCenterClient';
```

Comment:
We probably don't need the new /unit directory to differentiate tests which do not need VS Code to run.
People may get confused by this, and that tests inside this directory are real unit tests.
If we're removing this, we should probably also remove the new related instructions from the README.